src/SoCreate.ServiceFabric.PubSub/SoCreate.ServiceFabric.PubSub.csproj

```diff
@@ -29,19 +28,14 @@
     <PublicSign Condition=" '$(OS)' != 'Windows_NT' ">true</PublicSign>
```

Comment:
Remove this: 
`  <PropertyGroup Condition=" '$(Configuration)' == 'Release'">
    <PublicSign Condition=" '$(OS)' != 'Windows_NT' ">true</PublicSign>
    <DelaySign>False</DelaySign>
    <SignAssembly>true</SignAssembly>
    <AssemblyOriginatorKeyFile>..\SoCreate.ServiceFabric.PubSub.snk</AssemblyOriginatorKeyFile>
  </PropertyGroup>
`